package.json

```diff
@@ -49,6 +56,10 @@
     },
     "scripts": {
         "test": "node tests/all-tests.js",
-        "start": "node gpii.js"
+        "start": "node gpii.js",
+        "browserify:http": "node node_modules/browserify/bin/cmd.js -s http node_modules/http-browserify/index.js -o browserify/http.js ",
+        "browserify:https": "node node_modules/browserify/bin/cmd.js -s https node_modules/https-browserify/index.js -o browserify/https.js ",
+        "browserify:url": "node node_modules/browserify/bin/cmd.js -s urlModule node_modules/url/url.js -o browserify/urlModule.js ",
+        "postinstall": "grunt browserifyPrep && npm run browserify:http && npm run browserify:https && npm run browserify:url"
```

Comment:
I have a fear that this step will fail when universal is deployed as a child dependency of another project - e.g. the windows repository - since postinstall runs after every install, but the dev dependencies are only installed for a top-level project.

It's a shame that the way our repositories are set up that our CI cannot check such issues automatically - it will be another benefit of moving over to the "monorepo" pattern. Could you please check manually by checking out the current master of windows, binding its dependency on universal to this branch, that its install and tests will run ok.

We'll in time need production-level tests for the platform repositories against CouchDB - I see that there is this JIRA https://issues.gpii.net/browse/GPII-1987 as part of the planning for this - are there others?